app/helpers/sessions_helper.rb

```diff
@@ -7,7 +7,12 @@ def current_user
     if (user_id = session[:user_id])
       @current_user ||= User.find_by(id: user_id)
     elsif (user_id = cookies.signed[:user_id])
-      @current_user = user_from_remember_token(user_id)
+      user = User.find_by(id: user_id)
+
+      if user&.authenticated?(:remember, cookies[:remember_token])
+        log_in user
+        @current_user = user
+      end
```

Comment:
```suggestion
      @current_user = user_from_remember_token(user_id)
```
Sao lại sửa code chỗ này, code đang gọn mà